catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -586,6 +537,20 @@ private void buildFeatureFilters(List<FeatureTypeType> featureTypes, List<String
         "Wfs Source {}: Number of validated Features = {}", getId(), featureTypeFilters.size());
   }
 
+  private MetacardMapper getMetacardMapper(final QName featureTypeName) {
+    return metacardMappers
```

Comment:
A slight improvement on readability would be to create a local variable that represents the filter. Ex. `Predicate<MetacardMapper> doesFeatureNameMatch = (mapper) -> mapper.getFeatureType().equals(featureTypeName.toString());`. Just a suggestion.

Reply:
Done.